.github/workflows/gradle.yml

```diff
@@ -0,0 +1,87 @@
+# Definition of GitHub Actions.
+# Designed to working both line's and forked repository.
+#
+# Note:
+#  VM Spec = 2core, 7G RAM.
+#  Set -xmx2g for Gracle Process. Rests used by JUnit processes.
+name: CI
+on: [push, pull_request]
+jobs:
+  test:
+    name: test (JDK ${{ matrix.java }})
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [ '1.8', '11.x' ] # LTS's. (As of 2019-10, Gradle don't support JDK 13.)
+    steps:
+    - name: actions/setup-java@v1 (JDK ${{ matrix.java }})
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+        architecture: x64
+    - uses: actions/checkout@v1
+    - run: |
+        mkdir -p ~/.gradle
+        echo "org.gradle.jvmargs=-Xms2g -Xmx2g" >> ~/.gradle/gradle.properties
```

Comment:
Q. Does `gradle.properties` file in this repo does not work? How about putting `org.gradle.jvmargs=-Xms2g -Xmx2g` and `org.gradle.daemon=false` to gradle.properties in this repo?